README.md

```diff
@@ -72,10 +46,12 @@ Additionally, the extension will monitor **Package.swift** and **Package.resolve
 
 > **Note**: When browsing the files in a package, Visual Studio Code may also open these files in the Explorer. If this is undesirable, open **Preferences ▸ Settings** and set **Explorer: Auto Reveal** to `false`.
 
-## Configuration
+### Debugging
+
+The Swift extension uses the [CoreLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb) extension for debugging.
```

Comment:
Maybe we should explicitly mention that this is also how users can run executables? It may seem obvious, but it may not be for everyone.